photon-api/src/main/scala/com/linkedin/photon/ml/model/RandomEffectModel.scala

```diff
@@ -234,19 +213,16 @@ object RandomEffectModel {
    *       that type - it will be faster for large numbers of random effect models. Note that it may still be a
```

Comment:
Remove the TODO comments.

Reply:
We spoke offline - I explained the meaning of the TODO comments